Extension/src/common.ts

```diff
@@ -301,12 +329,15 @@ export function resolveCachePath(input: string, additionalEnvironment: {[key: st
     return resolvedPath;
 }
 
-export function resolveVariables(input: string, additionalEnvironment: {[key: string]: string | string[]}): string {
+export function resolveVariables(input: string | undefined, additionalEnvironment: undefined | {[key: string]: string | string[]}): string {
     if (!input) {
         return "";
     }
+    let additionalEnvironment2: {[key: string]: string | string[]};
```

Comment:
I thought this shouldn't be necessary.  TypeScript should be able to promote away the undefined typing after it is assigned.  Does that not work?

Reply:
Unfortunately, the promotion does not extend into a lamba.  :/  Will move the promotion into the lambda.